### DISCUSSION TITLE
How to configure a basic LEMP + Mailhog env?

### DISCUSSION THREAD

**Discussion by diazwatson** [wardenenv/warden#686] [Help & Support]
Hi, I am trying to configure a basic LEMP env to work with a basic php application.

I initially tried the Laravel env as it contains most of the things I need but I get a 404. I assume it is because the structure of my project is different to a Laravel project. I also tried the local env but didn't know how to configure the .yml file into the .warden directory.

Is there a better way to do this?

Thanks

**Comment by navarr** [comment]
I haven't done this in a while, but it should be able the same.  I believe this only works if you have a single entrypoint router.

For me, I use the Laravel env type.  I then update `.warden/warden-env.yml` like such:

```
version: "3.5"
services:
  nginx:
    environment:
      - NGINX_PUBLIC=/pub
```

This is specifically for Yii, but in that case the index.php router is in the pub/ directory.  

If you're not using a single-entry router, another solution might be necessary

**Reply by diazwatson** [reply]
@navarr thanks for taking the time to answer.

Yes, my entry point is index.php, but unfortunately the Laravel env type with the  `.warden/warden-env.yml` didn't work.

What worked was the Wordpress env type which in essence (just noticed that), contains pretty much all I need:
```
warden env-init projectname wordpress
```
https://docs.warden.dev/environments/types.html#wordpress
